### ISSUE TITLE
Issue proper exception if not properly authenticated (404)

### ISSUE THREAD

**Issue by berndruecker** [camunda-community-hub/camunda-operate-client-java#11]
If not properly authenticated - a call to Operate leads to 404 (unauthenticated). This leads to the following exception when processing the response of the call:

```
java.lang.NullPointerException: Cannot invoke “com.fasterxml.jackson.databind.JsonNode.asText()” because the return value of “com.fasterxml.jackson.databind.JsonNode.get(String)
```

We need to inspect the HTTP Resonse code - and whenever this is 4041/403 [don't recall - need to retest] (or probably even whenever this is not 200?) throw a proper exception.

As this might be a regular problem, it is pretty bad that for the user it looks like a bug :-)

**Comment by chDame** [comment]
https://github.com/camunda-community-hub/camunda-operate-client-java/pull/12

**Comment by chDame** [comment]
resolved in 1.3.5
